src/InlineAutocomplete/utils.ts

```diff
@@ -2,8 +2,8 @@ import {Children, EventHandler, SyntheticEvent} from 'react'
 
 import {ShowSuggestionsEvent, Suggestion, TextInputCompatibleChild, Trigger} from './types'
 
-const singleWordTriggerTerminators = new Set([' ', '\n'])
-const multiWordTriggerTerminators = new Set(['.', '\n'])
+const singleWordTriggerTerminators = new Set([' ', '\n', '\r\n'])
```

Comment:
not sure these are needed but it looks safe to me + no tests are failing

Reply:
It technically shouldn't be necessary because I think this issue only shows up with templates when they are written offline in external editors. That said, it doesn't hurt to add it.